web/src/main/angular/src/app/core/components/removable-agent-list/removable-agent-list-container.component.ts

```diff
@@ -76,9 +76,9 @@ export class RemovableAgentListContainerComponent implements OnInit, OnDestroy {
                 map((data: IAgentList) => {
                     return Object.entries(data).reduce((acc: {[key: string]: any}[], [key, value]: [string, IAgent[]]) => {
                         return [...acc, ...value.map((agent: IAgent) => {
-                            const {applicationName, hostName, agentId, agentVersion, startTimestamp, ip} = agent;
-
-                            return {applicationName, hostName, agentId, agentVersion, startTimestamp, ip};
+                            const {applicationName, hostName, agentId, agentName, agentVersion, startTimestamp, ip} = agent;
+                            const agentNameText = agentName ? agentName : "N/A";
+                            return {applicationName, hostName, agentId, agentNameText, agentVersion, startTimestamp, ip};
```

Comment:
- Please use single quotation marks when using strings.
- Give an empty line after declaring a variable

```suggestion
                            const agentNameText = agentName ? agentName : 'N/A';
                            
                            return {applicationName, hostName, agentId, agentNameText, agentVersion, startTimestamp, ip};
```